patches/third_party-blink-renderer-build-scripts-core-css-properties-templates-style_builder_functions.tmpl.patch

```diff
@@ -1,31 +0,0 @@
-diff --git a/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl b/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
-index 5ff29d8618a2261fcf92c9a04471d21ecef5b8af..17c2fb60f35793c30899505b7185d790dc026023 100644
---- a/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
-+++ b/third_party/blink/renderer/build/scripts/core/css/properties/templates/style_builder_functions.tmpl
-@@ -18,7 +18,7 @@ void {{prefix ~ '::' if prefix}}ApplyValue(StyleResolverState& state, const CSSV
-     {% if property.style_builder_inline == header and property.style_builder_generate_initial %}
- {{declare_initial(property.name.to_upper_camel_case() if not header)|indent(spaces, true)}} {
- {{caller(property)|indent(spaces, true)-}}
--{{'}'|indent(spaces, true)}}
-+{{'\n}'|indent(spaces, true)}}
```

Comment:
In C74 it has been changed to:
<pre>
    {% if property.style_builder_inline == header and property.style_builder_generate_initial %}
{{declare_initial(property.name.to_upper_camel_case() if not header)|indent(spaces, true)}} {
{{(caller(property) ~ '}')|indent(spaces, true)}}
</pre>